test/resizing.jl

```diff
@@ -47,17 +47,23 @@ end
             test_imresize_interface(img, (5,10), (5,))
             test_imresize_interface(img, (5,10), 1:5) # FIXME: @inferred failed
             test_imresize_interface(img, (5,10), (1:5,)) # FIXME: @inferred failed
+            test_imresize_interface(img, (5,5), (5,5), CenterPoint(1, 1))
+            test_imresize_interface(img, (20,20), CenterPoint(1, 1), ratio = 2)
+            test_imresize_interface(img, (20,10), CenterPoint(1, 1), ratio = (2, 1))
```

Comment:
This only makes sure the function gets called and outputs something. We also need to test if the output is correct, for example:

- test if `imresize(img, (20,20), CenterPoint(1, 1))` works the same as `imresize(img, (20,20))`
- test if `imresize(img, (20, 20), CenterPoint(5, 5); method=Constant())[5, 5] == img[5, 5]` (If I'm right about the implementation, I'm not sure if it's `[5, 5]` here).
- If it's not supported, test if `imresize(imag, (20, 20), CenterPoint(-10, -10))` errors.

Also, `OffsetArray` is used widely in the ecosystem, so it would be great to also test `OffsetArray(img, -1, -1)` works normally for the above test cases.

Reply:
About testing if the `CenterPoint` doesn't change, without overriding the value it can be actully _quite_ different, so either we should override it or it won't be ensured. Except this point I'm working on it.

Reply:
`BSpline(Constant())` is the nearest interpolation, I believe this doesn't change the value?

Reply:
Yes, that's right.